routes/user.js

```diff
@@ -0,0 +1,15 @@
+import { Router } from 'express';
+import controllers from '../controllers';
+
+const router = Router();
+
+router.get('/', controllers.getLimitedUsersByLoginSubstring);
+
+router.route('/:id')
+    .get(controllers.getUserById)
+    .put(controllers.editUser)
+    .delete(controllers.deleteUser);
+
+router.post('/', controllers.createUser);
+
+export default router;
```

Comment:
Why do you need this file if already do this configuration with validation applied in `routes/validatedUser.js`?

Reply:
@Naooki Because there are 2 tasks. First one without validation and second one with validation. I also think that this is an artificial construction. ))